packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -60,12 +62,16 @@ function NotificationItem({
       )}
     >
       {onClick && (
-        <button
-          type="button"
-          aria-label="Open notification"
-          className="absolute inset-0 z-0"
-          onClick={onClick}
-        />
+        <Link href={targetUrl} passHref>
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/anchor-has-content, jsx-a11y/no-static-element-interactions */}
```

Comment:
Ideally we wouldn't have these, but I see some we can't fix.
Also was wondering if we would prefer the `Button` with `tag=a` or not.
Not really sure when we make the switch now.

Reply:
I tried that, but the our button component would need more modifications to it to make it work out of the box as this does work. So that's why I didn't use it. Though not sure either when we are supposed to use that 🤔

Reply:
Agree on using a plain element since there won't be any content/UI to this. Just the disabling of eslint if we can trim it down. I remember this was the exact reason why we used the button but didn't realize yet the impact on the right click.

Reply:
I think you forgot to clean up the disabled eslints before merging.

Reply:
Oh sorry, didn't get that right. So we want to fix the eslints so we won't disable them? All are required at the moment with the current implementation. 
We could fix the key events one by adding a key event handler.

Reply:
I think we might need the doubleClick handler which the button already handled.
It might cut down that ESlint.

The others I think we can't fix and have multiple issues with in different places.